ide-integration/integration-with-visual-studio.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Integration with Visual Studio
+page_title: Integration with Visual Studio
```

Comment:
Consider different 'title', 'page_title', 'description' and 'first #' for SEO purposes